starboard/build/config/base_configuration.gni

```diff
@@ -162,6 +162,12 @@ declare_args() {
   # Enables an NPLB audit of C++17 support.
   sb_enable_cpp17_audit = true
 
+  # Enables an NPLB audit of C++20 support.
+  sb_enable_cpp20_audit = true
+
+  # Flag to use a future version of Skia, currently not available.
+  use_skia_next = false
```

Comment:
Nit: I just removed this flag from configs in main, lets not reintroduce it by accident :)

Reply:
oops, removed in the last commit